cudax/include/cuda/experimental/__execution/cpos.cuh

```diff
@@ -37,13 +37,13 @@ template <__disposition _Disposition>
 struct __completion_tag
 {
   template <__disposition _OtherDisposition>
-  _CCCL_NODEBUG_API constexpr auto operator==(__completion_tag<_OtherDisposition>) const noexcept -> bool
+  _CCCL_TRIVIAL_API constexpr auto operator==(__completion_tag<_OtherDisposition>) const noexcept -> bool
```

Comment:
I do not understand the consequences of this change.

Reply:
@ericniebler wrote:
> these annotations make debugging easier and shorten stack traces to just the good parts.

We're gonna put these macros back like they were after the nightlies run, right?

Reply:
Ah, got it.  Thanks for explaining!